### PR TITLE
Set up basic views

### PR DESCRIPTION
**Pull request by MartyTheeMartian**
* Navbar
* Homepage
* Dashboard

### CODE REVIEW

**Comment by MartyTheeMartian** [comment]
@elimisteve Latest Pursuance PR

File: public/index.html [hunk 1]
```diff
@@ -1,40 +1,22 @@
...
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
 
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/latest/js/bootstrap.min.js"></script>
```
Comment (elimisteve) [inline public/index.html]: @MartyTheeMartian For Pursuance we'll have very secure HTTP headers, just like LeapChat does, that prevents the browser from fetching and executing JavaScript from another domain.  So please bundle `react-bootstrap` etc using `npm`/`package.json` :-)

File: src/App.js [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+import React, { Component } from 'react'
+import NavBar from './components/NavBar/NavBar';
+import Footer from './components/Footer/Footer';
+import HomePage from './components/HomePage/HomePage';
+import Dashboard from './components/Dashboard/Dashboard';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Route,
+} from 'react-router-dom';;
```
Comment (elimisteve) [inline src/App.js]: @MartyTheeMartian `;;`

File: src/components/HomePage/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+              <i className="fa fa-long-arrow-right" aria-hidden="true"></i>
+            </div>
+            <div>
+              <h3>Collaborate</h3>
+            </div>
+            <div>
+              <i className="fa fa-long-arrow-right" aria-hidden="true"></i>
+            </div>
+            <div>
+              <h3>Justice</h3>
```
Comment (elimisteve) [inline src/components/HomePage/HomePage.js]: @MartyTheeMartian https://keybase.pub/pursuance/images/

File: public/index.html (hunk 1 above)
Comment (elimisteve) [reply public/index.html]: (If this requires a WebPack clusterfuck, don't worry about it for now)

**Comment by MartyTheeMartian** [comment]
@elimisteve closing this PR and creating a new one

**Pull request closed without merging**